assets/custom.js

```diff
@@ -243,4 +243,23 @@ $('.remove').click(function(){
             console.log('error', error);
         }
     })
+});
+
+//Display Footer sub-menu when footer menu is clicked
+$('.footer__links-heading').click(function(){
+    const chevron = $(this).find('.fa-chevron-down');
+    const submenu = $(this).next();
+    const scrollHeight = submenu.prop('scrollHeight');
+    const expandedHeight = scrollHeight + "px";
+    const mediaQuery = window.matchMedia('(max-width: 767px'); //Allows footer menu heading to be clicked ONLY on mobile viewports
+    if (mediaQuery.matches) {
+        $(this).toggleClass('active');
+        if ($(this).hasClass('active')) {
+            submenu.css('maxHeight', expandedHeight);
+            chevron.css('transform', 'rotateX(180deg)');
+        } else {
+            submenu.css('maxHeight', '0')
+            chevron.css('transform', 'rotateX(0deg)');
```

Comment:
i think this can be achieved through css: `.footer__links-heading.active` 
(remember, always do styling in CSS if possible)

also, not sure why you would need to set `max-height`. you should be able to achieve the accordion without setting it but let me know if otherwise.

Reply:
ah i see, you're doing it for the animation. gotcha, nevermind then